themes/bootstrap5/templates/RecordTab/holdingsils/standard.phtml

```diff
@@ -4,6 +4,17 @@
     $checkStorageRetrievalRequest = $holding['checkStorageRetrievalRequest'] ?? false;
     $checkILLRequest = $holding['checkILLRequest'] ?? false;
     $availabilityStatus = $holding['availability'];
+    $getThisEnabled = isset($this->getThis) && (!isset($this->driver) || $this->driver->supportsGetThis());
```

Comment:
Do you want an unset driver to default to true, or false? It looks like the later code depends on `$this->driver` being set, so I'm wondering if this is a logic error and should be revised to:

```suggestion
    $getThisEnabled = isset($this->getThis) && $this->driver?->supportsGetThis();
```